MonkeyBot/Migrations/20211005163642_UpdatedReminders.cs

```diff
@@ -7,52 +7,48 @@ public partial class UpdatedReminders : Migration
-                columns: table => new
-                {
-                    Id = table.Column<int>(type: "INTEGER", nullable: false)
-                        .Annotation("Sqlite:Autoincrement", true),
-                    GuildId = table.Column<ulong>(type: "INTEGER", nullable: false),
-                    ChannelId = table.Column<ulong>(type: "INTEGER", nullable: false),
-                    Type = table.Column<string>(type: "TEXT", nullable: false),
-                    Name = table.Column<string>(type: "TEXT", nullable: false),
-                    Message = table.Column<string>(type: "TEXT", nullable: false),
-                    ExecutionTime = table.Column<DateTime>(type: "TEXT", nullable: true),
-                    CronExpression = table.Column<string>(type: "TEXT", nullable: true)
-                },
-                constraints: table =>
-                {
-                    table.PrimaryKey("PK_Reminder", x => x.Id);
-                });
+            migrationBuilder.RenameTable(
+                name: "Announcements",
+                newName: "Reminder");
+
+            migrationBuilder.RenameColumn(
+                table: "Reminder",
+                name: "ID",
+                newName: "Id");
+
+            migrationBuilder.RenameColumn(
+                table: "Reminder",
+                name: "GuildID",
+                newName: "GuildId");
+
+            migrationBuilder.RenameColumn(
+                table: "Reminder",
+                name: "ChannelID",
+                newName: "ChannelId");
         }
 
         protected override void Down(MigrationBuilder migrationBuilder)
         {
-            migrationBuilder.DropTable(
-                name: "Reminder");
 
-            migrationBuilder.CreateTable(
-                name: "Announcements",
-                columns: table => new
-                {
-                    ID = table.Column<int>(type: "INTEGER", nullable: false)
-                        .Annotation("Sqlite:Autoincrement", true),
-                    ChannelID = table.Column<ulong>(type: "INTEGER", nullable: false),
-                    CronExpression = table.Column<string>(type: "TEXT", nullable: true),
-                    ExecutionTime = table.Column<DateTime>(type: "TEXT", nullable: true),
-                    GuildID = table.Column<ulong>(type: "INTEGER", nullable: false),
-                    Message = table.Column<string>(type: "TEXT", nullable: false),
-                    Name = table.Column<string>(type: "TEXT", nullable: false),
-                    Type = table.Column<string>(type: "TEXT", nullable: false)
-                },
-                constraints: table =>
-                {
-                    table.PrimaryKey("PK_Announcements", x => x.ID);
-                });
+            migrationBuilder.RenameTable(
+                name: "Reminder",
+                newName: "Announcements");
+
+            migrationBuilder.RenameColumn(
+                table: "Reminder",
```

Comment:
table name should be Announcements here or the RenameTable needs to be the last step.